### ISSUE TITLE
Rework which practice exercises should be listed in the concept tree

### ISSUE THREAD

**Issue by sanderploegsma** [exercism/java#2631]
Currently the [concept tree on the Java track](https://exercism.org/tracks/java/concepts) is very hard to follow due to the large number of exercises listed for each concept. The reason this is happening is because all practice exercises list a number of concepts in their `practices` configuration. This probably started when migrating to Exercism v3.

The problem with this is that the concepts listed for each exercise only mean that that exercise is _related_ to that concept. There is no way that there are 33 exercises to help a student practice the `strings` concept... Instead, every concept should contain a few _hand-picked_ practice exercises that _actually help a student to practice that concept_. For the `strings` concept for example, some candidates are `isogram` and `isbn-verifier`.

So, the goal of this issue is to arrive at a concept tree that contains a relevant handful of practice exercises for each concept. A good example of this is the [concept tree on the Python track](https://exercism.org/tracks/python/concepts).

**Comment by NagaNitya** [comment]
Hi, is this still an issue? because now when I look at the Java concepts tree, it lists only 1 exercise per concept...

**Comment by SMSRZ** [comment]
Hi, is this still an issue ?

**Comment by Cwengii** [comment]
Hi, is this issue still open?
